app/controllers/ems_block_storage_controller.rb

```diff
@@ -0,0 +1,43 @@
+class EmsBlockStorageController < ApplicationController
```

Comment:
@martinpovolny just curious - what is that supposed to do - the menu code clearly works for me without the line in both the EmsBlockStorageController and EmsObjectStorageController - neither of which have the line.  Thanks.